src/Fixer/PhpUnit/PhpUnitMethodCasingFixer.php

```diff
@@ -0,0 +1,277 @@
+        foreach ($phpUnitTestCaseIndicator->findPhpUnitClasses($tokens) as $indexes) {
+            $this->applyCasing($tokens, $indexes[0], $indexes[1]);
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function createConfigurationDefinition()
+    {
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('case', 'Apply camel or snake case to test methods'))
+                ->setAllowedValues([self::CAMEL_CASE, self::SNAKE_CASE])
+                ->setDefault(self::CAMEL_CASE)
+                ->getOption(),
+        ]);
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $startIndex
+     * @param int    $endIndex
+     */
+    private function applyCasing(Tokens $tokens, $startIndex, $endIndex)
+    {
+        for ($index = $endIndex - 1; $index > $startIndex; --$index) {
+            if (!$this->isTestMethod($tokens, $index)) {
+                continue;
+            }
+
+            $functionNameIndex = $tokens->getNextMeaningfulToken($index);
+            $functionName = $tokens[$functionNameIndex]->getContent();
+            $newFunctionName = $this->updateMethodCasing($functionName);
+
+            if ($newFunctionName !== $functionName) {
+                $tokens[$functionNameIndex] = new Token([T_STRING, $newFunctionName]);
+            }
+
+            $docBlockIndex = $this->getDocBlockIndex($tokens, $index);
+            if ($this->hasDocBlock($tokens, $index)) {
+                $this->updateDocBlock($tokens, $docBlockIndex);
+            }
+        }
+    }
+
+    /**
+     * @param string $functionName
+     *
+     * @return string
+     */
+    private function updateMethodCasing($functionName)
+    {
+        if (self::CAMEL_CASE === $this->configuration['case']) {
+            $newFunctionName = $functionName;
+            $newFunctionName = str_replace('_', ' ', $newFunctionName);
+            $newFunctionName = ucwords($newFunctionName);
+            $newFunctionName = str_replace(' ', '', $newFunctionName);
+            $newFunctionName = lcfirst($newFunctionName);
+        } else {
+            $newFunctionName = Utils::camelCaseToUnderscore($functionName);
+        }
+
+        return $newFunctionName;
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int$index
+     *
+     * @return bool
+     */
+    private function isTestMethod(Tokens $tokens, $index)
+    {
+        // Check if we are dealing with a (non abstract, non lambda) function
+        if (!$this->isMethod($tokens, $index)) {
+            return false;
+        }
+
+        // if the function name starts with test its a test
+        $functionNameIndex = $tokens->getNextMeaningfulToken($index);
+        $functionName = $tokens[$functionNameIndex]->getContent();
+
+        if ($this->startsWith('test', $functionName)) {
+            return true;
+        }
+        // If the function doesn't have test in its name, and no doc block, its not a test
```

Comment:
1st `its` is correct, 2nd isn't -> https://data.grammarbook.com/blog/pronouns/1-grammar-error/